remix-app/app/routes/_index.tsx

```diff
@@ -49,9 +42,34 @@ export const loader = async () => {
     return {
       id: task.id,
       title: task.title,
-      desc: task.desc,
-      status: task.status
-    }
+      status: task.status,
+      updatedAt: task.updatedAt,
+    };
   });
-  return json({ tasks: data })
+  return { tasks: data };
+}
+
+const dateToStr = (input: Date): string => {
+  const result = new Date(input).toLocaleDateString("ja-JP", {year: "numeric",month: "2-digit", day: "2-digit"});
+  return result;
+};
+
+const taskItem = (tasks: TaskType[], status: string) => {
+  return (
+    <div className='border rounded-lg p-4 my-4 flex-1'>
+      {status}
+      {tasks.map((task) => (
+        task.status === status ? (
```

Comment:
```suggestion
        task.status === status && (
```
これすれば、3項演算子にしなくても良いかもしれません！

Reply:
修正しました！

https://github.com/Sawawa42/crud-with-remix/pull/14/commits/504d746212bac9ebe17edd92d51983cb41154939